lib/index.js

```diff
@@ -58,6 +58,8 @@ api.db = null;
 // shared collections cache
 api.collections = {};
 
+// Note: api.writeOptions is deprecated
+// and will be removed in release 9.0
```

Comment:
This sort of thing should perhaps be noted in the changelog, maybe detected and logged, and probably needs an issue so someone remember to remove it.

Reply:
PR for that here and good catch: https://github.com/digitalbazaar/bedrock-mongodb/pull/66